### ISSUE TITLE
Shredder: size limits don't work correctly

### ISSUE THREAD

**Issue by hungrywolf27** [sahib/rmlint#335]
The size limits in the settings page of the Shredder GUI don't work correctly. For example, a lower limit of 1 Kilobyte in the GUI is passed as `0M` in the command line, shown in the debug output. The problem seems to be with constructing the command line options here:
https://github.com/sahib/rmlint/blob/8a2fde263de72fc4d009688b53e30123a3f96efb/gui/shredder/runner.py#L176-L182

Is there any reason not to pass the values simply as bytes? This works for me:

```python3
        min_size, max_size = cfg.get_value('traverse-size-limits')
        extra_options += [
            '--size', '{a}-{b}'.format(
                a=min_size,
                b=max_size
            )
        ]
```

**Comment by sahib** [comment]
Thanks for the report.

> Is there any reason not to pass the values simply as bytes? This works for me:

Not a good one at least. I remember some silly integer overflow issue when saving a very large integer into the config. This was a temporary workaround that I forgot to clean up. Will do soon.

**Comment by sahib** [comment]
Could not reproduce the issue I was talking about. Fixed in the way you proposed as of 5a5cf895df405ae4eb3d4d9dd87d5c76de66dbbc.
